### DISCUSSION TITLE
Add full resolved files URL

### DISCUSSION THREAD

**Discussion by lluishi93** [directus/directus#3182] [Feature Requests]
In Direcus 8, the File interface fields returned the full src url. However, in Directus 9, it is not like this anymore:

```
{
  "data": {
    "id": 1,
    "name": "Logo",
    "blank": true,
    "image": {
      "id": "f15aa370-7ae5-47b6-a56a-1ad804017c70",
      "storage": "s3",
      "filename_disk": "f15aa370-7ae5-47b6-a56a-1ad804017c70.svg",
      "filename_download": "hotel-one.svg",
      "title": "Hotel One",
      "type": "image/svg+xml",
      "uploaded_on": "2020-11-24T08:01:33Z",
      "modified_by": null,
      "modified_on": "2020-11-24T08:01:34Z",
      "charset": null,
      "filesize": 4365,
      "width": null,
      "height": null,
      "duration": null,
      "embed": null,
      "description": null,
      "location": null,
      "tags": null,
      "metadata": null
    }
  }
}
```

Could it be possible to add a field with the full resolved url? 
`https://servername.com/assets/fileId`

**Comment by rijkvanzanten** [comment]
You already know the API URL because you're hitting the API to get this info in the first place, so what is preventing you from doing "URL + /assets/ + id" ?

**Reply by lluishi93** [reply]
Just pure convenience, in order to easily transition from previous Directus 8 installations.

**Reply by botzill** [reply]
I agree that having a full url, with token of the current user would be easier

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
